test/metabase/channel/render/card_test.clj

```diff
@@ -206,31 +207,41 @@
     (mt/dataset test-data
-      (mt/with-temp [:model/Card {base-card-id :id} {:dataset_query {:database (mt/id)
-                                                                     :type     :query
-                                                                     :query    {:source-table (mt/id :orders)
-                                                                                :expressions  {"Tax Rate" [:/
-                                                                                                           [:field (mt/id :orders :tax) {:base-type :type/Float}]
-                                                                                                           [:field (mt/id :orders :total) {:base-type :type/Float}]]},
-                                                                                :fields       [[:field (mt/id :orders :tax) {:base-type :type/Float}]
-                                                                                               [:field (mt/id :orders :total) {:base-type :type/Float}]
-                                                                                               [:expression "Tax Rate"]]
-                                                                                :limit        10}}}
+      (mt/with-temp [:model/Card {base-card-id :id
+                                  base-query :dataset_query} {:dataset_query
+                                                              {:database (mt/id)
+                                                               :type     :query
+                                                               :query    {:source-table (mt/id :orders)
+                                                                          :expressions  {"Tax Rate" [:/
+                                                                                                     [:field (mt/id :orders :tax) {:base-type :type/Float}]
+                                                                                                     [:field (mt/id :orders :total) {:base-type :type/Float}]]},
+                                                                          :fields       [[:field (mt/id :orders :tax) {:base-type :type/Float}]
+                                                                                         [:field (mt/id :orders :total) {:base-type :type/Float}]
+                                                                                         [:expression "Tax Rate"]]
+                                                                          :limit        10}}}
                      :model/Card {model-card-id  :id
                                   model-query    :dataset_query
                                   model-metadata :result_metadata
                                   :as            model-card} {:type            :model
+                                                              :entity_id       "zQ1h5VDeOchxwMDYSK7om"
```

Comment:
I would prefer naming `"zQ1h5VDeOchxwMDYSK7om"`, e.g., as `entity-id`. Generating a random one would be the clearest, IMO.